test/test_geometry.py

```diff
@@ -470,7 +477,7 @@ def test_populate_bounds_antimeridian(self):
 
         # Convert everything into radians
         vertices_rad = np.radians(vertices_lonlat)
-        vertices_cart = [node_lonlat_rad_to_xyz(v) for v in vertices_rad]
+        vertices_cart = np.vstack([_lonlat_rad_to_xyz(vertices_rad[:, 0], vertices_rad[:, 1])]).T
```

Comment:
@erogluorhan 

I've updated the ones that could be easily vectorized. Some of the others would require significant re-writes of the test cases, which is beyond the scope of this PR, and many of them are iterating over very small numbers of points (i.e. 4, 5, 6), so performance is not a concern.

Reply:
That sounds great! Please open an issue for them for future reference even if we will cover them together with the face bounds optimization in a single PR. Regardless, I am approving this now. Great work!